app/src/lib/stores/app-store.ts

```diff
@@ -4688,6 +4717,21 @@ export class AppStore extends TypedBaseStore<IAppState> {
     await createDesktopStashEntry(repository, branchName, tip.branch.tip.sha)
   }
 
+  public async _popStash(repository: Repository, branch: Branch) {
+    const stash = await getLastDesktopStashEntryForBranch(
+      repository,
+      branch.name
+    )
+
+    if (stash === null) {
+      log.warn('no stash found that matches')
+      return
+    }
+
+    await popStashEntry(repository, stash.stashSha)
+    log.warn(`Popped stash with commit id ${stash.stashSha}`)
```

Comment:
why is this `warn` and not `info` or `debug`? doesn't seem like its something we should be even mildly worried about.

Reply:
Yea, I'll change to `info`. My main concern is that what ever is logged is actually persisted to disk.